src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/DependencyAnalysisState.cs

```diff
@@ -6,12 +6,22 @@
 
 namespace Microsoft.DotNet.UpgradeAssistant.Steps.Packages
 {
-    internal class DependencyAnalysisState : IDependencyAnalysisState
+    public class DependencyAnalysisState : IDependencyAnalysisState
```

Comment:
Does this need to be made public?

Reply:
based on offline chat, leaving this as is